Sources/AppStoreConnectCLI/Commands/TestFlight/BetaGroups/Sync/SyncBetaGroupsCommand.swift

```diff
@@ -0,0 +1,17 @@
+// Copyright 2020 Itty Bitty Apps Pty Ltd
+
+import ArgumentParser
+
+struct SyncBetaGroupsCommand: ParsableCommand {
+    static var configuration = CommandConfiguration(
+        commandName: "sync",
+        abstract: """
+        Sync information about beta groups with provided configuration file.
+        """,
```

Comment:
This doesn't need to be a multi-line literal.
```suggestion
        abstract: "Sync information about beta groups with provided configuration file.",
```